03-transactions.md

```diff
@@ -117,17 +117,76 @@ If a revoked commitment transaction is published, the other party can spend this
 
 #### `to_remote` Output
 
-This output sends funds to the other peer and thus is a simple P2WPKH to `remotepubkey`.
+This output sends funds to the other peer, thus is not encumbered by a
+revocation private key.
+
+If `option_anchor_outputs` applies to the commitment transaction, the `to_remote` output is delayed similarly to the `to_local` output, and is to a fixed key:
+
+    `max_to_self_delay`
```

Comment:
If we're changing this script entirely, I believe @ariard explained to use that you don't need to put that CSV constrain in the script when it's not in a conditional branch, it can simply be on the `nSequence` field itself (and the script remains just a P2WPKH). 
Can you confirm @ariard ?

Reply:
That's only the case when you've created an off-chain covenant via multi-sig like we have for our second layer HTLCs. If the CSV isn't in the script, then the relative time-lock is actually optional if multi-sig isn't used.

Reply:
I'm re-reading BIP 68, and it says that the `nSequence` can be used as a relative timelock without any script (and always enforced if the right flags are set).
What BIP 112 adds is the ability to enforce the relative timelock only on specific conditional branches of your script.

> If the CSV isn't in the script, then the relative time-lock is actually optional if multi-sig isn't used.

Where does that come from? I don't see anything about that in BIP 68 or BIP 112...
It seems to me that either no-one is leveraging BIP 68, or I'm missing maybe another BIP or code change that changed the implementation's behavior regarding `nSequence` :confused:

Reply:
@t-bast `nSequence` is always to be enforced by the base layer but if spent output is only locked by pubkey of one party, it has control of spending transaction construction and so can always set value of field to 0 to sweep out the locktime requirement. So for contract semantic being safe, you need to enforce the locktime in the `scriptPubkey` spent, assuming this one is also committed by all parties.

fyi, OP_CHECKTEMPLATEVERIFY (but not the current bip version) or Taproot annex per-input locktime may alleviate this in a far future.

Reply:
Aah gotcha I got confused there.